comm/index.html

```diff
@@ -84,9 +85,85 @@
             </li>
             <li><a href="http://dir.gmane.org/gmane.comp.gis.geoserver.devel">Alternate archive</a> from GMane</li>
         </ul>
+       <p> The
+            same <a href="userlist-guidelines.html" class="alert-link">post guidelines</a> as the user list apply to the developer list.</p> 
     </section>
 </section>
 
+<section>
+    <h2>Issue Tracker</h2>
+    <p>
+      If you have a problem when working with GeoServer, then please let us know through the mailing lists. GeoServer uses <a href="https://osgeo-org.atlassian.net/projects/GEOS">JIRA</a>, a bug tracking website, to manage issue reports.
```

Comment:
I wanted to ask about "please let us know through the mailing lists". 

Was that intentional to cut down on duplicate bug reports? Or should we say issue tracker there...